library/src/main/java/com/novoda/downloadmanager/MigrationJob.java

```diff
@@ -183,39 +184,4 @@ private void migrateCompleteDownloads(InternalMigrationStatus migrationStatus,
         onUpdate(migrationStatus);
     }
 
-    private void migrateV1FilesToV2Location(InternalFilePersistence internalFilePersistence, Migration migration) {
```

Comment:
nice, this might have improved the performance/time done for migrations, do you have any results on how long does it takes now?

Reply:
It's pretty much instant now 😂